core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -127,7 +128,12 @@ private Optimizely(@Nonnull EventHandler eventHandler,
             if (getProjectConfig() != null) {
                 updateODPSettings();
             }
-            addUpdateConfigNotificationHandler(configNotification -> { updateODPSettings(); });
+            if (projectConfigManager != null) {
+                NotificationRegistry.getNotificationCenter(projectConfigManager.getSDKKey()).
```

Comment:
This can be confusing with with public NotificationCenter. What about changing names to 
`NotificationRegistry.getInternalNotificationCenter` or some other names to avoid confusion?
@msohailhussain What do you think?